### PR TITLE
Draft edits for Schema.org and RRM transformations

### PR DESCRIPTION
**Pull request by sohkai**
Should be reviewed last, and will likely be split up into two parts so the pretty diff can work.

TODO: 

* [x] Rebase on top of all other changes
* [x] grep for `data-structure/` or `schema/` and update as necessary with #13 
* [x] Run through spell check again

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -448,56 +449,50 @@ Potentially, these could be used later to help define the COALA IP specification
...
+In the context of COALA IP, any application-agnostic schemata, including all [schemata derived from
+the LCC RRM](#coala-ip-implementing-the-lcc-rrm-with-linked-data), would become a *hosted*
+extension. Fortunately, leveraging schema.org in this way maintains compliance with rules five and
+six of the [LCC's "Ten Targets"](#the-lccs-ten-targets):
 
-- Rule 5: Links between identifiers are system agnostic and need to be authorized by participating
+- **Rule 5**: Links between identifiers are system agnostic and must be authorized by participating
   consortiums.
-- Rule 6: Metadata is system agnostic and its schema has to be authorized by participating parties
+- **Rule 6**: Metadata is system agnostic and its schema must be authorized by participating parties
```
Comment (TimDaub) [inline README.md]: Both Rule 5 and 6 don't match the framing in the upper section #the-lccs-ten-targets.

My suggestion would be to either:

- copy exact framing from above
- remove repetition of rules

File: README.md [hunk 2]
```diff
@@ -1448,10 +1455,10 @@ satisfy the consolidated requirements of:
...
 
 
-RRM Rights are just links between `Manifestation`s and licenses. Licenses are usually intended to be
-read by humans, so the license should be stored on a technology that prevents changes to the
-license. This could be an immutable ledger or blockchain, or be linked by Content-Addressing. With
-this in mind, an implementation in IPLD/IPFS is favored:
+The `Right` can be seen as the link between a `Manifestation` and its licenses. To prevent
+undetected changes to these linked licenses, the licenses would ideally be stored on an immutable
+ledger or content-addressed storage layer. With this in mind, the implementation in IPLD (on IPFS)
+is favoured:
```
Comment (TimDaub) [inline README.md]: We removed "Licenses are usually intended to be read by humans".
I think this statement should still be included somehow.

**Review by TimDaub — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sohkai) [reply README.md]: Nice catch; I've opted to copy the framing from above. See f8dbfb5934d53967fab2d3c41e1e021b576e9436.

File: README.md (hunk 2 above)
Comment (sohkai) [reply README.md]: See f8dbfb5934d53967fab2d3c41e1e021b576e9436.

**Comment by sohkai** [comment]
@TimDaub Not sure if I left this unmerged before I left, or if it was approved after, but I fixed the merge conflict and I'll leave you to choose what to do with it 💃 :).

**Pull request merged**